dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -1,27 +1,27 @@
 # For testing and analyzing code.
-mypy==0.701.*
-pylint==2.3.*
-pytest==3.8.*
-pytest-cov==2.5.*
-pytest-benchmark==3.2.*
-yapf==0.27.*
+mypy~=0.701
+pylint~=2.3
+pytest~=3.8
+pytest-cov~=2.5
+pytest-benchmark~=3.2
+yapf~=0.27
 
 # For generating protobufs
-grpcio-tools==1.21.*
+grpcio-tools~=1.21
 mypy-protobuf==1.10
 
 # For uploading packages to pypi.
 twine
 
 # For verifying behavior of qasm output.
-qiskit==0.6.*
+qiskit~=0.6
 
 # For generating documentation.
 pypandoc
-recommonmark >= 0.4.0
+recommonmark ~= 0.4.0
```

Comment:
This limits the allowed version to [0.4.0, 0.5) instead of [0.4.0, inf).  Is this OK for this package?

Reply:
moved it back, though I think `>=` is considered dangerous.